### PR TITLE
selinux: Add a permissive boolean

### LINKED ISSUE #1269: SELinux permissive policies (alternative to LOG action of seccomp)
#### What would you like to be added:
A way to mark a SELinux policy as permissive without resorting to `RawSelinuxProfile`.

#### Why is this needed:
In case the recording misses some calls or in general if the admin is not sure that the policy can be put into production

#### User story covered
As an application SRE, Rajesh needs to be able to do long runs of a security profile in “complain-mode” to easily identify the impact of a profile being enforced without impacting the application.

#### Implementation notes
Probably just need to set `typepermissive` based on a flag.

#### Acceptance criteria:
 - the profile will be "not enforced", but will log the actions to audit.log. This can be tested by removing parts of the profile and making sure the app still works, but the actions are logged

### PR DESCRIPTION
**Pull request by jhrozek**
#### What type of PR is this?
/kind feature
/kind api-change

#### What this PR does / why we need it:
Similarly to how seccomp profiles allow setting the default action to
LOG, let's allow the selinux profiles to be set to permissive mode. This
way, all calls would have still be allowed, but would appear in
audit.log.

This is useful in case the user is just iterating on a policy and wants
to deploy a first version of the policy in the wild without causing AVC
denials.

#### Which issue(s) this PR fixes:
Fixes #1269

#### Does this PR have test?
Yes

#### Special notes for your reviewer:
N/A

#### Does this PR introduce a user-facing change?

```release-note
SELinux profiles gained a new attribute .spec.permissive which defaults to false. When set to true, the profile will
run in a permissive mode, that means that all actions would be allowed, but logged. This allows for a more iterative
approach for profile development.
```

### CODE REVIEW

**Review by ccojocar — APPROVED** [review]

File: installation-usage.md
Comment (jhrozek) [reply installation-usage.md:852]: oops, sorry. Fixed.

**Review by JAORMX — APPROVED** [review]
/approve

LGTM. Let's wait for CI before merging. Thanks for adding this!

**Comment by jhrozek** [comment]
this should fix the verify issue..

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**
